examples/pipeline_wavernn/mol_loss.py

```diff
@@ -0,0 +1,89 @@
+import torch
+import torch.nn.functional as F
+
+
+def log_sum_exp(x):
+    """ numerically stable log_sum_exp implementation that prevents overflow """
+
+    axis = len(x.size()) - 1
+    m, _ = torch.max(x, dim=axis)
+    m2, _ = torch.max(x, dim=axis, keepdim=True)
+    return m + torch.log(torch.sum(torch.exp(x - m2), dim=axis))
+
+
+def Mol_Loss(y_hat, y, num_classes=65536, log_scale_min=None, reduce=True):
```

Comment:
nit: one word?

Reply:
Now I use `MoLLoss` as the function name and `losses` as the file name.